DOCUMENTATION.md

```diff
@@ -0,0 +1,103 @@
+## Table of Contents
+* [Description](#description)
+* [MQTTClient](#mqttclient)
+* [connect](#connect)
+* [close](#close)
+* [set_uplink_callback](#set_uplink_callback)
+  * [uplink_callback](#uplink_callback)
+* [set_connect_callback](#set_connect_callback)
+  * [connect_callback](#connect_callback)
+* [set_downlink_callback](#set_downlink_callback)
+  * [downlink_callback](#downlink_callback)
+* [set_close_callback](#set_close_callback)
+  * [close_callback](#close_callback)
+* [send](#send)
+* [License](#license)
+
+## Description
+
+This package provides you an easy way to connect to The Things Network via MQTT.
+
+### MQTTClient
+
+The class constructor can be called following this scheme:
+```python
+MQTTClient(app_id, app_access_key, [mqtt_address], [discovery_address])
+```
+- `app_id`: **string**  this the name given to the application when it was created.
+![Screenshot of the console with app section](./images/app-console.png?raw=true)
+- `app_access_key`: **string**  this can be found at the bottom of the application page under **ACCESS KEYS**.
+![Screenshot of the console with accesskey section](./images/accesskey-console.png?raw=true)
+- `mqtt_address`: **string**  this  is the address of the handler to which the application was registered. It needs to be provided as an `mqtt_address=value` argument when calling the constructor.
+- `discovery_address`: **string** this is the address of the discovery server to use in order to find back the address of the MQTT handler. It needs to be provided as an `discovery_address=value` argument when calling the constructor.
+The constructor returns an **MQTTClient object** set up with the application informations, connected to The Things Network.
+
+### connect
+Connects and starts the client in the background. This function also re-established the client's connection in case it was closed.
+```python
+client.connect()
+```
+
+### close
+Disconnects and stop the client from which the method is called.
+```python
+client.close()
+```
+
+### Using Callbacks
+
+The callback functions are functions which are executed when a trigger event happens.
+
+#### set_uplink_callback
+Set the callback function, to be called when an uplink message is received.
+```python
+client.set_uplink_callback(uplink_callback)
+```
+
+##### uplink_callback
+The callback function must be declared in your script following this structure:
+* `uplink_callback(msg, client)`
+  * `msg`: **JSON ob**  the message received by the client
+  * `client`: **object**  the client from which the callback is executed are calling
+
+#### set_connect_callback
+Set the connection callback function to be executed when the client connect to the broker.
```

Comment:
when the client connect**s**

Reply:
Can you also make it clear whether only one connect callback function will be executed, or whether you can add more?